README.md

```diff
@@ -92,6 +95,27 @@ $ ./bin/upload-ccd-spreadsheet.sh -v
 #### Login into CCD
 Open management web page http://localhost:3451 and login with user created above
 
+#### Troubleshooting
+
+##### Start CCD Web script fails on first run
+
+IdAM API takes long time to boot up.
+This causes incomplete docker setup.
+Inspect `docker ps -a` idam container, wait for completion, then run `./bin/start-ccd-web.sh` again.
+In all occasions never experienced a failure afterwards
+
+##### CCD Web is up and running, but cannot log in
+
+There can be multiple reasons including core breaking changes introduced by services enlisted in `docker-compose.yml` file.
+
+First course of actions is to check whether CCD definition got imported successfully as it is creating user profiles which are mandatory for login
+
+The following are suggestions of possible culprits:
+
+- role mismatch between idam and ccd. Check both `ccd-importer` and `idam-importer`
```

Comment:
you mean container logs here I assume ?

Reply:
This whole section is about local ccd web setup in docker. Of course it'll be about containers/images